src/KintoBase.js

```diff
@@ -1,7 +1,5 @@
-"use strict";
-
```

Comment:
out of curiosity why did you remove `use strict`? is it useless because of babel? #noob ;)